setup.py

```diff
@@ -118,17 +118,7 @@ def run(self):
       author='Roberto Alsina and others',
       author_email='ralsina@netmanagers.com.ar',
       url='https://getnikola.com/',
-      packages=['nikola',
-                'nikola.plugins',
-                'nikola.plugins.command',
-                'nikola.plugins.compile',
-                'nikola.plugins.compile.ipynb',
-                'nikola.plugins.compile.markdown',
-                'nikola.plugins.compile.rest',
-                'nikola.plugins.task',
-                'nikola.plugins.task.sitemap',
-                'nikola.plugins.template',
-                ],
+      packages=find_packages(),
```

Comment:
In fact, closing this one and opening a new PR in the getnikola/nikola becuase probably there are other things to fix...